scipy/special/_add_newdocs.py

```diff
@@ -11103,12 +11128,13 @@ def add_newdoc(name, doc):
 
     Parameters
     ----------
+    mn : scalar or ndarray
+        The mean of the normal distribution.
     p : array_like
         CDF values, in range (0, 1].
     x : array_like
         Quantiles, i.e. the upper limit of integration.
-    mn : scalar or ndarray
-        The mean of the normal distribution.
```

Comment:
The signature above says `nrdtrisd(p, x, mn, out=None)`, but here the order is `mn, `p`, `x`. I think these should be consistent.

Reply:
🤦 done